.gitignore

```diff
@@ -30,6 +30,7 @@ packages/contracts-bedrock/deployments/anvil
 .env
 .env*
 !.env.example
```

Comment:
Can we delete `.env.example` from this now?

Reply:
I guess, seems like a different PR tho